src/styles/index.scss

```diff
@@ -1,3 +1,112 @@
+@import "vars";
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: $text-second-color;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    color: $text-main-color;
+    font-weight: 700;
+  }
+
+  &__button {
+    height: 40px;
+    background-color: $main-accent;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 4px;
+    width: 100%;
+    color: #fff;
+    border: none;
+    text-transform: uppercase;
+    padding: 12px;
+    margin-top: 16px;
+    font-weight: 700;
+    box-sizing: border-box;
+
+    &:hover {
+      color: $main-accent;
+      background-color: #fff;
+      border: 1px solid $main-accent;
```

Comment:
Better add a transparent border for the default state of the element and just change its color on hover